src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -127,6 +127,8 @@ const Authenticate = (() => {
         const $files             = $not_authenticated.find('input[type="file"]');
         $button = $not_authenticated.find('#btn_submit');
 
+        hideSuccess();
```

Comment:
better to call it elsewhere, otherwise it gets called everywhere that wants to disable the button, what do you think?

Reply:
yeap i agree, i've updated the location :)